### ISSUE TITLE
Add CLS to metric graphs

### ISSUE THREAD

**Issue by patrickhulce** [GoogleChrome/lighthouse-ci#620]
**Describe the solution you'd like**
CLS has a major weight in the score, is a Core Web Vital, and should be visually trackable in the dashboard. No design was ever created for it given that it was added to Lighthouse after LHCI was a thing and it's on another scale compared to all the other metrics.

**Describe alternatives you've considered**
None.

**Comment by dkarzon** [comment]
I made a start on this but I wasn't able to get the project running locally yet 😞 

https://github.com/GoogleChrome/lighthouse-ci/compare/main...dkarzon:cls-graph

This should be the API and UI components of it so new builds will have the statistics for it generated and the graphs are added to the dashboard but I wasn't sure how to do the database migration part to get the statistics for existing builds and I haven't been able to test it.

**Comment by patrickhulce** [comment]
Thanks very much @dkarzon! I think the complexity here though is in how to design the dashboard to avoid the odd numbered graph issues while still visualizing on smaller desktop viewport sizes.

**Comment by vikasrohit** [comment]
@patrickhulce what do you mean by `odd numbered graph issues` here?

**Comment by patrickhulce** [comment]
> @patrickhulce what do you mean by odd numbered graph issues here?

Simply that we've got a nice two column layout going at the moment that fits nicely in a single viewport. If we just added a 3rd graph with CLS it would be thrown off a bit and require some scrolling. That's the simple solution here and given the lack of movement here maybe that's just the best course of action for now 🤷

**Comment by vikasrohit** [comment]
@patrickhulce I am able to get upto this point where I can render the graph for CLS and as first graph has multiple metrices rendered, I tried to move it down to occupy more space.
<img width="1428" alt="Screenshot 2022-04-18 at 11 49 09 AM" src="https://user-images.githubusercontent.com/2417632/163764644-dd44c230-b7be-4ae4-aac1-76e97c7eff94.png">

However, there seems to be an existing bug which prevents the tooltip (hover-card) from being rendered for the graph which is rendered last (in this screenshot graph with FCP/LCP/TTI/SI). If I move the CLS graph to the last, it stops rendering hover-card for it. I am still trying to debug it.

**Comment by vikasrohit** [comment]
@patrickhulce did you get chance to look at this layout?

**Comment by patrickhulce** [comment]
if we flip the FCP/LCP graph above the other two then LGTM :)

**Comment by vikasrohit** [comment]
> if we flip the FCP/LCP graph above the other two then LGTM :)

Yes, that also looks good and infact I tried that first. This layout I tried to see if there is bug already existing or is it introduced because of my changes. The bug I am talking about is:

>However, there seems to be an existing bug which prevents the tooltip (hover-card) from being rendered for the graph which is rendered last (in this screenshot graph with FCP/LCP/TTI/SI). If I move the CLS graph to the last, it stops rendering hover-card for it. I am still trying to debug it.

**Comment by vikasrohit** [comment]
@patrickhulce do we have any guide for how to fix unit tests for these changes? I am seeing some unit tests failing after introduction of new graph.

**Comment by patrickhulce** [comment]
I understand that, but I can't help unless I know _which_ tests and _how_ they are being broken :)

**Comment by vikasrohit** [comment]
@patrickhulce here are few of the tests (`snapshots` mostly) which are failing:
<img width="1417" alt="Screenshot 2022-05-24 at 10 38 02 AM" src="https://user-images.githubusercontent.com/2417632/169953487-3b94d139-308b-4705-9bd9-3b64ff35c8a8.png">
<img width="1420" alt="Screenshot 2022-05-24 at 10 39 16 AM" src="https://user-images.githubusercontent.com/2417632/169953555-187ff897-0b5e-4719-b8fe-f3a35729fe00.png">

**Comment by patrickhulce** [comment]
OK and have you opened the diff to see if what has changed is expected? If you're changing the UI, updating snapshots is normal :)

**Comment by vikasrohit** [comment]
I didn't try to open diff yet as I didn't realize that I can see the diff image as well.
On the second point, is there any guide how to update snapshots?

**Comment by tmotyl** [comment]
Same problem with missing "Interaction to Next Paint (INP)" this core web vital is also not rendered in the graph.
Are there any hints how to add it manually? Maybe some plugin?
